SU2_CFD/src/numerics/flow/flow_sources.cpp

```diff
@@ -329,6 +329,7 @@ CNumerics::ResidualType<> CSourceIncAxisymmetric_Flow::ComputeResidual(const CCo
                              TWO3*AuxVar_Grad_i[0][1]);
       residual[3] -= Volume*yinv*Thermal_Conductivity_i*PrimVar_Grad_i[nDim+1][1];
 
+      if (implicit) jacobian[2][2] = Volume*yinv*2.0*total_viscosity*yinv;
```

Comment:
Note that adding the jacobian of the energy equation in the incompressible flow will affect (residuals of) all incompressible testcases that have energy equation activated!

Reply:
Hmm, i am not sure that this is the root of that. This is in `CSourceIncAxisymmetric_Flow::ComputeResidual` which should only affect cases with axisymmetric flow. Also, the  Jacobian [2][2] is `d residual[2] / d U[2]` where U[2] is y-velocity, as far as I understand it. So the energy equation should not play a role here